tests/kernel/wave/wave_e2e_test.py

```diff
@@ -101,6 +101,7 @@ def test(
         },
         canonicalize=True,
         run=True,
+        run_bench=True,

```

Comment:
This should not be enabled by default, we need a separate option. Also, where bench results are saved?